apps/events/models/speaker.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Speaker(models.Model):
+    full_name = models.CharField(max_length=50)
+    photo = models.ImageField(null=True)     # Or we can use a URLField()
```

Comment:
Will be great to have this content uploaded to a particular path.
```suggestion
    photo = models.ImageField(null=True, upload_to="images/speakers")     # Or we can use a URLField()
```